### PR TITLE
steal.lic: Add data builder mode that removes redundant actions if yo…

### PR DESCRIPTION
**Pull request by Sarvatt**
…u are just updating the stealing database.

### CODE REVIEW

File: steal.lic [hunk 1]
```diff
@@ -35,7 +35,7 @@ class Steal
     @slow_bin_speed = settings.slow_bin_speed
 
     @difficulty_keys = %w[exceptionally very rather acceptable poorly vpoorly nearly trivial]
-
+    @data_builder_mode = settings.stealing_data_builder_mode
```
Comment (jandersson) [inline steal.lic]: If this is a new setting it should be added to base.yaml

Comment (jandersson) [reply steal.lic]: that was fast

File: steal.lic [hunk 2]
```diff
@@ -49,11 +49,11 @@ class Steal
     ]
 
     args = parse_args(arg_definitions)
-
+    @data_builder_mode_toggle = false
```
Comment (jandersson) [inline steal.lic]: When is this set to true?

Comment (jandersson) [reply steal.lic]: Is it the same as `@data_builder_mode`?

Comment (jandersson) [reply steal.lic]: disregard

File: steal.lic [hunk 3]
```diff
@@ -148,9 +148,12 @@ class Steal
   end
 
   def build_target_data(item, container)
+    @data_builder_mode_toggle = true if @data_builder_mode
```
Comment (jandersson) [inline steal.lic]: I think its confusing to have this buried in a function - is there a reason it cant be in setup?

**Comment by Sarvatt** [comment]
thanks for the feedback, I'm trying to push this for Byd and he's updated it with enough changes that I should close this until I get them all integrated.

**Pull request closed without merging**